src/shared/components/Types/TypesIcon.tsx

```diff
@@ -20,14 +20,14 @@ const TypesIcon: React.SFC<TypesIconProps> = ({ type }) => {
   }).toString();
   const src = `data:image/png;base64,${imageData}`;
   return (
-    <li className="types-icon">
```

Comment:
I had to change to Div, because it's quite possible that LI will exist within LI and thats not HTML5 compliant